doc/sphinxext/gen_rst.py

```diff
@@ -8,19 +8,19 @@
 Files that generate images should start with 'plot'
```

Comment:
I don't think you should bother about this file too much. We are supposed to switch to [sphinx-gallery](http://github.com/sphinx-gallery/sphinx-gallery) at one point and I think it is already compatible with Python3.